orchestration/dap_orchestration/pipelines.py

```diff
@@ -75,3 +77,44 @@ def refresh_data_all() -> None:
         eols_transform_records(eols_extract_records())
     ]
     write_outfiles(collected_outputs)
+
+
+@pipeline(
```

Comment:
I don't think we need a new pipeline. Dagster let's us run subsets of a pipeline via the `solid_selection` arg to this decorator. In this case we can give it `["sample_extract_records*"]`, which will run the "sample_extract_records and all solids downstream of it (i.e., sample_extract_records -> sample_transformation_records -> write_outfiles)